src/west/manifest.py

```diff
@@ -35,6 +35,11 @@
 #: A fully qualified reference to `MANIFEST_REV_BRANCH`.
 QUAL_MANIFEST_REV_BRANCH = 'refs/heads/' + MANIFEST_REV_BRANCH
 
+#: The west refs space used for current manifest revision.
+#: West ref space is temporary space for fetching of SHAs.
+#: The refspace also useable for future west operations, such as west rebase.
```

Comment:
Can you remove this line since we haven't agreed west rebase is the name of a feature we will for sure implement?

Reply:
I'm trying to hint what I would like to do when having a little spare time ;)
Bu agree, this comment can be removed.